src/app/modules/layout/navbar/navbar.component.ts

```diff
@@ -22,4 +22,9 @@ export class NavbarComponent implements OnInit {
   onLogout(): void {
     this.authService.logout();
   }
+
```

Comment:
Nice

Reply:
We can include this in Strings materials for OP course 👍